Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,87 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure prepares the telescope and dome to take flatfileds and describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
+This is the default end-of-checkout script.
+
+.. NOTE::
+   Only run the prepare to take flats if you have sufficient time to vent the telescope afterwards. It will leave the telescope and dome in position to start taking afternoon calibrations. This is az = 205.7 elev = 39. dome = 20.0
```

Comment:
I think here I will use the name of the script
``auxtel/prepare_for/flats.py`` instead of "prepare to take flats"